src/QGISStyleParser.ts

```diff
@@ -767,6 +913,87 @@ export class QGISStyleParser implements StyleParser {
     };
   }
 
+  convertTextSymbolizerRule(qmlRuleList: any[], rule: Rule) {
+    let textSymbolizer: TextSymbolizer;
+    rule.symbolizers.forEach(symbolizer => {
+      if (symbolizer.kind === 'Text') {
+        textSymbolizer = symbolizer as TextSymbolizer;
+        const textStyleAttributes: any = {
+          fontSize: textSymbolizer.size || 12,
+          fontLetterSpacing: textSymbolizer.letterSpacing || 0,
+          multilineHeight: textSymbolizer.lineHeight || 1,
+          textColor: textSymbolizer.color ? this.qmlColorFromHexAndOpacity(textSymbolizer.color, 1) : '0,0,0,255'
+        };
+        if (textSymbolizer.font) {
+          textStyleAttributes.fontFamily = textSymbolizer.font[0];
+        }
+        if (textSymbolizer.label) {
+          textStyleAttributes.fieldName = textSymbolizer.label.replace('{{', '').replace('}}', '');
```

Comment:
This will only work if `textSymbolizer.label` only contains a single variable name, e.g. `"{{area}}"`. We also allow mixtures of variables and static text, e.g. `"{{area}} sqm"`.

Reply:
True. I think we'd need to have a closer look at how qgis does this and then write a converter. At least this enables basic usage to work.